nano/templates/firealarm.tmpl

```diff
@@ -5,6 +5,9 @@ The current alert level is:
 {{if data.alertLevel == 'blue'}}
 	<span style='color: rgb(13, 186, 212);font-weight: bold;'>Blue</span>
 {{/if}}
+{{if data.alertLevel == 'yellow'}}
+	<span class='yellow'>Yellow</span>
```

Comment:
Is yellow even a class?

Reply:
you said yellow not red (which was bad)

Reply:
I wondered about that too, but thought, surely they checked...

Reply:
It's not.